core/state/snapshot/snapshot.go

```diff
@@ -405,9 +384,15 @@ func (t *Tree) Cap(root common.Hash, layers int) error {
 // layer limit is reached, memory cap is also enforced (but not before).
 //
 // The method returns the new disk layer if diffs were persisted into it.
+//
+// Note, the final diff layer count in general will be one more than the amount
+// requested. This happens because the bottommost diff layer is the acumulator
+// which may or may not overflow and cascade to disk. Since this last layer's
```

Comment:
```suggestion
// which may or may not overflow and cascade to disk. Since this last layers
```

Reply:
It's possessive form, `last layer's survival`, as in "the survival of the last layer".